extensions/amp-carousel/0.1/base-slides.js

```diff
@@ -196,4 +206,18 @@ export class BaseSlides extends BaseCarousel {
       this.autoplayTimeoutId_ = null;
     }
   }
+
+  /**
+  * Remove autoplay.
+  * @protected
+  */
+  removeAutoplay() {
+    this.clearAutoplay();
+    this.hasAutoplay_ = false;
+    this.shouldAutoplay_ = this.hasAutoplay_ && this.isLoopingEligible();
+    this.element.removeAttribute('loop');
```

Comment:
we should not disable `loop` unless it was forced by autoplay in the first place. If author has `loop` specified, after autoplay ends, user should still manually loop.